rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+UTXO exchanges, ensuring that transaction details remain confidential. The sequencer verifies these exchanges without accessing specific details, enhancing privacy by unlinking 
+sender and receiver identities. Zero-Knowledge Proofs (ZKPs) allow users to prove transaction validity without revealing data, maintaining the integrity and confidentiality of 
+private transactions. UTXOs contain assets such as balances, NFTs, or private storage data, and are stored in plaintext within Sparse Merkle Trees (SMTs) in the private state and 
+as hashes in the public state. This dual-storage approach keeps UTXO details confidential while allowing public verification through hashes, achieving a balance between privacy and transparency.
+
+
+Implementing NSSA introduces unique challenges, particularly in cryptographic implementation and maintaining the integrity of private executions. These challenges are addressed 
+through various solutions such as ZKPs, which ensure transaction validity without compromising privacy, and the dual-storage approach, which maintains confidentiality while enabling 
+public verification. By allowing users to customize their privacy settings, NSSA enhances user experience and promotes wider adoption of private execution platforms. As we move towards 
+a future where user-empowered privacy control is crucial, NSSA provides a flexible and user-centric solution that meets the diverse needs of blockchain users.
+
+### Why NSSA is different than other execution hybrid platforms
+
+In many existing hybrid execution platforms, privacy settings are predefined by developers, often applying a one-size-fits-all approach that does not accommodate the 
+diverse privacy needs of users. These platforms blend public and private states, but the control over privacy remains with the application developers. 
+While this approach is straightforward for developers, who bear the responsibility for any potential privacy leaks, it leaves users with no control over their own privacy settings. 
+This rigidity becomes problematic as user needs evolve over time, or as new regulations necessitate changes to privacy configurations. In such cases, 
+updates to decentralized applications are required to adjust privacy settings, which can disrupt the user experience and create friction.
+
+Nescience State Separation architecture addresses these limitations by introducing a groundbreaking concept: **selective privacy**. Unlike traditional platforms where privacy 
+is static and determined by developers, selective privacy empowers users to dynamically choose their own privacy levels based on their unique needs and sensitivity. 
+This flexibility is critical in a decentralized ecosystem where the diversity of users and use cases demands a more adaptable privacy solution.
+
+In the NSSA model, users have the autonomy to select how they interact with decentralized applications (Dapps) by choosing from four types of transaction executions: **public, 
```

Comment:
Replace: 
"Dapps"

with:
"dapps"

(this is an industry-specific word, admittedly and there are many stylisations. However, the linguistics department of comms has discussed it at some length and we believe we can help normalise its usage in language by formating it as a regular noun. It is also how Ethereum has started stylising it.